Tests/SwiftSyntaxBuilderTest/VariableTests.swift

```diff
@@ -25,7 +25,7 @@ final class VariableTests: XCTestCase {
     }
 
     let syntax = buildable.buildSyntax(format: Format(), leadingTrivia: leadingTrivia)
-    XCTAssertEqual(syntax.description, "␣var d: [String: Int] = [:]")
+    XCTAssertEqual(syntax.description, "␣var d: [String: Int] = [: ]")
```

Comment:
The empty dictionary literal is now generated as `[: ]` since the colon token seems to require a trailing space. I don't think this should be an issue, but perhaps worth pointing out.

Reply:
You should be able to fix it in the [`DictionaryExprConvenienceInitializers.swift`](https://github.com/apple/swift-syntax/blob/0ec7b5a7f96aa652aac910767b4e4371f6cda488/Sources/SwiftSyntaxBuilder/DictionaryExprConvenienceInitializers.swift#L26)

Reply:
Good catch, thanks!